packages/openapi-parser/src/configuration/index.ts

```diff
@@ -29,6 +29,7 @@ export const ERRORS = {
   EXTERNAL_REFERENCE_NOT_FOUND: 'Can’t resolve external reference: %s',
   FILE_DOES_NOT_EXIST: 'File does not exist: %s',
   NO_CONTENT: 'No content found',
+  INVALID_CONTENT: 'Unable to parse valid JSON or YAML.',
```

Comment:
nit, wdyt about using:

```suggestion
  INVALID_CONTENT: 'Failed to parse content as valid JSON or YAML.',
```

Reply:
should we have one error per type at some point?